app/helpers/omniauth_callbacks_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```

Comment:
You can remove empty helpers files. 
Also, you can configure rails generator to skip generate helpers